### PR TITLE
change the gtk package deps for build errror and fix cabal file error

### PR DESCRIPTION
**Pull request by mut0u**

### CODE REVIEW

**Comment by phuhl** [comment]
@mut0u I appreciate the effort, but it still does not build. Any idea why?

**Comment by GuessWhatBBQ** [comment]
Can confirm that cloning @mut0u's fork and trying to compile works just fine.

After poking around the `deadd-notification-center.cabal` file I also came to the same conclusion that a primary issue with compiling was the fact that the build-depends was not updated when the executable was renamed. Changing the build-depends should solve the issue with stack complaining about not finding a linux-notification-center package, which it did on my local machine but the github CI seems to still show that error meaning it's not actually using the patched cabal file.

TL;DR, this patch fixes the compile issues
